### PR TITLE
Set primary key on classifications table

### PR DESCRIPTION
**Pull request by Nate-Wessel**
## What this pull request accomplishes:

- creates a primary key on the table

## Issue(s) this solves:

- lets other tables reference this key

## What, in particular, needs to reviewed:

- check syntax

### CODE REVIEW

**Comment by Nate-Wessel** [comment]
@chmnata I think we did this already? Is this good to go?

File: volumes/miovision/sql/create-table-classifications.sql [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
     classification text COLLATE pg_catalog."default",
     location_only boolean, -- for peds and bikes, where movement isn't available, only which leg they were observed on
     class_type text COLLATE pg_catalog."default"
 )
 WITH (
     OIDS = FALSE
 )
-TABLESPACE pg_default;
\ No newline at end of file
+TABLESPACE pg_default;
```
Comment (chmnata) [inline volumes/miovision/sql/create-table-classifications.sql:11]: can we add a  `Alter table ... owner to miovision_admins; ` ? Realized it was missing from the original file.

Comment (Nate-Wessel) [reply volumes/miovision/sql/create-table-classifications.sql:11]: Done!

**Review by chmnata — APPROVED** [review]

**Comment by Nate-Wessel** [comment]
@chmnata are you authorized to merge this?

**Pull request merged**